pkg/snapshotterutil/sociutil.go

```diff
@@ -47,7 +50,65 @@ func CreateSoci(rawRef string, gOpts types.GlobalCommandOptions, allPlatform boo
 	if gOpts.Namespace != "" {
 		sociCmd.Args = append(sociCmd.Args, "--namespace", gOpts.Namespace)
 	}
-	// #endregion
+
+	return sociCmd, nil
+}
+
+// ConvertSociIndexV2 converts an image to SOCI format and returns the converted image reference with digest
+func ConvertSociIndexV2(ctx context.Context, client *client.Client, srcRef string, destRef string, gOpts types.GlobalCommandOptions, platforms []string, sOpts types.SociOptions) (string, error) {
+	sociCmd, err := setupSociCommand(gOpts)
```

Comment:
This will rely on the platform SOCI version to support the convert option as well. For compatibility, should we check the SOCI cli version first, before making the soci call. We should throw an appropriate error/warn message if the platform soci version does not support convert option.